README.md

```diff
@@ -1,54 +1,316 @@
+WOVN.php requires PHP 5.3 or higher. WOVN.php has no third-party dependencies.
+Depending on your configuration, you might have to install and/or activate the
+PHP module `mod_rewrite` (see [Section 2.3.2.](#232-for-static-websites) and
+[Section 3.2.](#32-optional-parameters)).
 
-After pasting, your directory structure should look like the following:
+WOVN.php has been tested with Apache 2 and Nginx. We provide installation
+instructions for both. If you use other technologies, we encourage you to
+[contact us](mailto:support@wovn.io) for support.
+
+## 2. Getting Started
+### 2.1. Download WOVN.php
+To install WOVN.php, you have to manually download WOVN.php from our Github
+repository. The root directory of WOVN.php must be place  at the root of your
+website's directory. In the rest of this document, we consider the root
+directory of your website to be `/website/root/directory`.
+```
+$ cd /website/root/directory
+$ wget https://github.com/WOVNio/WOVN.php/archive/master.zip -O WOVN.php.zip
+$ unzip WOVN.php.zip; mv WOVN.php-master WOVN.php
+```
+
+**Note on updates:** When you need to update WOVN.php, you can simply replace
+all the content of `WOVN.php` directory by the content of the new version.
+
+### 2.2. Basic configuration
+In order for WOVN.php to work with your WOVN.io project, you need to fill a
+configuration file. The configuration file must be named `wovn.ini` and be
+placed at the root of your website's directory. You can start from the sample
+file at `WOVN.php/wovn.ini.sample`.
+```
+$ cp WOVN.php/wovn.ini.sample wovn.ini
+```
+
+In this section, we give you the basic configuration you should use to get
+started. You can find complete details of WOVN.php configuration at
+[Section 3.](#3-configuration). To get started, you need to know at least your
+WOVN.io project token, the original language of your website and the languages
+your website can be translated into by WOVN.io. To obtain your project token,
+you can visit your project dashboard, click on "INTEGRATION METHODS" and then
+select the "PHP Library" installation method.
+
+Bellow is an example of `wovn.ini` for a project with token "TOKEN", original
+language English (`en`) and translated languages Japanese (`ja`) and French
+(`fr`).
+```
+project_token = TOKEN
+url_pattern_name = query
+default_lang = en
+supported_langs[] = ja
+supported_langs[] = fr
+```
+
+At the end of this stage, the file structure of you website should look like
+below.
+```
++ /website/root/directory
+  + WOVN.php
+  - wovn.ini
+  [...]
+```
+
+### 2.3. Activate WOVN.php
+In order for WOVN.php to localize your website, you need to activate content
+interception. There are two activation methods depending on how you Web pages
+are generated. If your Web pages are generated by PHP files, please follow the
+[instructions for dynamic websites](#231-for-dynamic-websites). If your Web
+pages are pure HTML, please follow the
+[instructions for static websites](#232-for-static-websites).
```

Comment:
Since the instructions sections are immediately below, I get a bit confused by these links. Maybe it's ok to just say

> If your web pages are generated by PHP files, please follow the instructions for dynamic websites below. If your web pages are pure HTML, please use the instructions for static websites.

(I also removed capitalization of "web")

Reply:
All done, thanks